pyccel/ast/builtins.py

```diff
@@ -354,6 +355,48 @@ def arg(self):
     def __str__(self):
         return 'float({0})'.format(str(self.arg))
 
+# ===========================================================================
+class PythonRound(PyccelAstNode):
+    """ Represents a call to Python's native round() function.
+    """
+    __slots__ = ('_arg', '_ndigits', '_dtype', '_precision')
+    name = 'round'
+    _rank = 0
+    _shape = ()
+    _order = None
+    _attribute_nodes  = ('_arg','_ndigits')
+
+    def __init__(self, number, ndigits = None):
```

Comment:
`self._ndigits` could be simply written as `self._ndigits = ndigits`

Reply:
You're right. It's left over from a moment when I was assigning `0` to `self._ndigits`.

Fixed [af5f147](https://github.com/pyccel/pyccel/pull/996/commits/af5f14731a4a2a14f757e2dc94c18738c889037a)